app/src/main/java/com/google/android/samples/socialite/ui/videoedit/VideoEditScreenViewModel.kt

```diff
@@ -67,14 +65,19 @@ class VideoEditScreenViewModel @Inject constructor(
     private val _isProcessing = MutableStateFlow(false)
     val isProcessing: StateFlow<Boolean> = _isProcessing
 
+    private var _videoSaveState = MutableStateFlow(VideoSaveState.PENDING)
+    val videoSaveState: SharedFlow<VideoSaveState> = _videoSaveState
```

Comment:
```suggestion
    val videoSaveState: StateFlow<VideoSaveState> = _videoSaveState.asStateFlow()
```

Reply:
Done (minus the call to `asStateFlow()` to match the rest of the code)